shoes-core/lib/shoes/oval.rb

```diff
@@ -10,9 +10,9 @@ class Oval
     STYLES = { fill: Shoes::COLORS[:black] }
 
     def create_dimensions(left, top, width, height)
-      left   ||= @style[:left]
-      top    ||= @style[:top]
-      width  ||= @style[:diameter] || @style[:width] || (@style[:radius] || 0) * 2
+      left ||= @style[:left]
+      top ||= @style[:top]
+      width ||= @style[:diameter] || @style[:width] || (@style[:radius] || 0) * 2
```

Comment:
hm, I do disagree with this - might be my personal style (and that my IDE makes those for me) but I like lining op `=` operators just as I like lining up `=>` in hashes, tremendously improves readability imo :)